src/payment/strategies/cba-mpgs/cba-mpgs.ts

```diff
@@ -0,0 +1,56 @@
+export const THREE_D_SECURE_PROCEED = 'PROCEED';
+export const THREE_D_SECURE_BUSY = 'SERVER_BUSY';
+export const THREE_D_SECURE_AVAILABLE = 'AUTHENTICATION_AVAILABLE';
+
+export interface CBAMPGSHostWindow extends Window {
+  ThreeDS?: ThreeDSjs;
```

Comment:
Is this an optional property for `CBAMPGSHostWindow` window?

Reply:
Yes, this property references the JS client of MPGS which before loading the script its actually undefined, so it can be used to validate if the script was loaded successfully by the script-loader